bin/dev

```diff
@@ -72,6 +74,22 @@ HEREDOC
     args.unshift("docker-compose", "-f /usr/local/dev-env/docker/dev-env.yml", "-p devenv")
     system(args.join(" "))
   end
+
+  def webinject(args)
+    args = ["/bin/bash", "-l"] if args.empty?
+    container_id = `docker container ls -q -f name=ermanagement_web_`.strip
+    if container_id == ""
+      puts "Container is not running"
+    else
+      system("docker exec -i -t #{container_id} #{args.join(" ")}")
+    end
+  end
+
+  def cmd(args)
+    args = ["/bin/bash", "-l"] if args.empty?
```

Comment:
Just curious... what does `-l` do?

Reply:
i think it was something i had to add to get rvm to work, but i don't think these containers are using rvm so we can prolly remove it.